crosstests/google-java/src/test/kotlin/build/buf/connect/crosstest/CrossTest.kt

```diff
@@ -71,12 +75,27 @@ class CrossTest(
                 NetworkProtocol.GRPC
             )
         }
+
+        @JvmField
+        @ClassRule
+        val CROSSTEST_CONTAINER = GenericContainer("bufbuild/connect-crosstest:${CROSSTEST_VERSION}")
+            .withExposedPorts(8080, 8081)
+            .withCommand(
+                "/usr/local/bin/serverconnect",
+                "--h1port",
+                "8080",
+                "--h2port",
+                "8081",
+                "--cert",
+                "cert/localhost.crt",
+                "--key",
+                "cert/localhost.key",
+            )
     }
 
     @Before
     fun before() {
-        val port = 8081
-        val host = "https://localhost:$port"
+        val host = "https://localhost:${CROSSTEST_CONTAINER.getMappedPort(8081)}"
```

Comment:
This will use an ephemeral port so it won't fail if something else is bound to 8080/8081.